paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodVerticalLayoutUITest.kt

```diff
@@ -33,7 +33,7 @@ import org.robolectric.annotation.Config
 @Config(sdk = [Build.VERSION_CODES.Q])
 @OptIn(ExperimentalEmbeddedPaymentElementApi::class)
 internal class PaymentMethodVerticalLayoutUITest(
```

Comment:
Should we rename this class? Can we rename it so it's easy to know it tests both?

Reply:
Updated to `PaymentMethodLayoutUITest`